libs/shared-types/src/lib/TpJobseekerProfile.ts

```diff
@@ -50,6 +50,7 @@ export type TpJobseekerProfile = {
   hrSummit2021JobFairCompanyJobPreferences?: HrSummit2021JobFairCompanyJobPreferenceRecord[]
 
   isProfileVisibleToCompanies: boolean
+  isSubscribedToTPMarketingEmails: boolean
```

Comment:
Please check if things work if you remove this

Reply:
Yes, it works. Removing it.